Orange/widgets/model/owneuralnetwork.py

```diff
@@ -0,0 +1,87 @@
+import re
+from AnyQt.QtCore import Qt
+
+from Orange.data import Table
+from Orange.modelling import NNLearner
+from Orange.widgets import gui
+from Orange.widgets.settings import Setting
+from Orange.widgets.utils.owlearnerwidget import OWBaseLearner
+
+
+class OWNNLearner(OWBaseLearner):
+    name = "Neural Network"
+    description = "Predict according to the nearest training instances."
+    icon = "icons/NN.svg"
+    priority = 90
+
+    LEARNER = NNLearner
+
+    activation = ["identity", "logistic", "tanh", "relu"]
+    solver = ["lbfgs", "sgd", "adam"]
+
+    learner_name = Setting("Neural Network")
+    hidden_layers_input = Setting("100")
+    activation_index = Setting(3)
+    solver_index = Setting(2)
+    alpha = Setting(0.0001)
+    max_iterations = Setting(200)
+
+    def add_main_layout(self):
+        box = gui.vBox(self.controlArea, "Network")
+        self.hidden_layers_edit = gui.lineEdit(box, self,
+                                               "hidden_layers_input",
+                                               label="Hidden layers:",
+                                               orientation=Qt.Horizontal,
+                                               callback=self.settings_changed)
+        self.activation_combo = gui.comboBox(
+            box, self, "activation_index", orientation=Qt.Horizontal,
+            label="Activation:", items=[i.capitalize() for i in self.activation],
```

Comment:
The correct human-friendly labels would be: Identity, Logistic, tanh and ReLU. Imho `.upper()` would server better.